### PR TITLE
Add notes for 2019-06-21 org-wide project manager convo

### PR DESCRIPTION
**Pull request by patcon**

### CODE REVIEW

File: 2019-06-21-org-wide-pm.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+	- what are the things we want to **follow through** on in the next few months?
+    - the things we experiment with are **not commitments forever**
+    - things we experiment with, we should be able to move away
+    - the things we’re experimenting with organizational values and objectives
+    - example:
+        - we want to try trello
+        - exportability. lock-in escape.
+        - we should be able to take a risk. we have little to lose on choosing something like trello.
+        - reminds of **value proposition** (from Pivotal): that products can work on mainstream clouds or your own. recognizing many diff types of business and many diff types of challenges.
+- patcon: **open source tools are NOT always the best ones**. our asset is **trustworthiness**, and sometimes we should introduce closed source tools if they t
```
Comment (benhylau) [inline 2019-06-21-org-wide-pm.md]: `they t`?

File: 2019-06-21-org-wide-pm.md [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+	            - gain deep **knowledge of challenges** from outputs of process
+	            - surprise in unexpected technological solution to soft human challenge. **open mind** to hear more.
+
+## Action Items
+
+- [x] Review and clean notes (patcon+garry)
+- [x] Double-check with Garry whether we want to arrive with a draft doc (patcon)
+    - We don't. Go in for convo.
+- [x] ~~Write draft role proposal~~
+- [ ] Prepare summary in agenda
```
Comment (benhylau) [inline 2019-06-21-org-wide-pm.md]: Are we good to merge or are we waiting on this item?

**Comment by benhylau** [comment]
@garrying hoping to merge this soon pls review. @patcon see comments.

File: 2019-06-21-org-wide-pm.md (hunk 1 above)
Comment (patcon) [reply 2019-06-21-org-wide-pm.md]:
```suggestion
- patcon: **open source tools are NOT always the best ones**. our asset is **trustworthiness**, and sometimes we should introduce closed source tools if these tools support the client best in their short term needs.
```

File: 2019-06-21-org-wide-pm.md (hunk 2 above)
Comment (patcon) [reply 2019-06-21-org-wide-pm.md]: We're good -- bolding served that purpose

Comment (patcon) [reply 2019-06-21-org-wide-pm.md]:
```suggestion
- [x] Prepare summary in agenda. (bolding accomplished this)
```

**Comment by patcon** [comment]
Fixed. LGTM. Waiting on @garrying

**Comment by benhylau** [comment]
@patcon @garrying I am merging this

**Pull request merged**